lib/middleware/permissions.js

```diff
@@ -1,15 +1,15 @@
-class UnauthorisedError extends Error {
+class ForbiddenError extends Error {
   constructor() {
-    super('Unauthorised');
-    this.status = 401;
+    super('Forbidden');
+    this.status = 403;
   }
 }
```

Comment:
Can we replace this with https://github.com/UKHomeOffice/asl-service/blob/master/errors/unauthorised.js

Reply:
👍